Bluepill-cli/Bluepill-cli/Simulator/SimulatorRunner.m

```diff
@@ -190,6 +217,43 @@ + (void)createDeviceWithConfig:(BPConfiguration *)config andName:(NSString *)dev
                        }];
 }
 
++ (SimDevice *)findDeviceWithConfig:(BPConfiguration *)config andDeviceID:(NSUUID *)deviceID {
```

Comment:
I couldn't see the benefit of passing NSError ** here. look at the code of  createDeviceWithConfig, which has completion:(void (^)(NSError *, SimDevice *))completion.  But findDeviceWithConfig doesn't have an async call with completion block. any benefit of create our own NSError object and return to the caller?